apps/meteor/client/components/UserAutoCompleteMultiple/UserAutoCompleteMultipleOptions.tsx

```diff
@@ -0,0 +1,31 @@
+import { Options } from '@rocket.chat/fuselage';
+import React, { forwardRef, ComponentProps, RefAttributes, ReactElement, Ref } from 'react';
+
+import { UserAutoCompleteOptionType } from './UserAutoCompleteMultipleFederated';
+import UserAutoCompleteMultipleOption from './UserAutoCompleteMultipleOption';
+
+type Options = Array<[UserAutoCompleteOptionType['username'], UserAutoCompleteOptionType]>;
+
+const renderOptions = (
```

Comment:
This is a strange function. Why to construct components like it?

Reply:
I believe @gabriellsh can help here :)